app/views/hyrax/base/_form_warning.html.erb

```diff
@@ -14,14 +14,15 @@
 <ul>
   <li class="radio">
     <label>
-      <%= f.radio_button :content_alert, t('hyrax.content_warning.content_alert') %>
+      <% @checked = (f.object.content_alert.blank? || (f.object.content_alert == t('hyrax.content_warning.content_alert'))) %>
+      <%= f.radio_button :content_alert, t('hyrax.content_warning.content_alert'), class='content_alert_box', checked: @checked ? 'checked' : '' %>
```

Comment:
You probably already tested this, but does the `checked` form option work w/ regular truthy values? or does it automatically just set it to whatever is passed in?
`checked: @checked` would be cleaner, or `checked: !!@checked` if need be, but might not be possible all together

Reply:
`<input ... checked="checked">` is the most correct, but not required (except for xHTML), `<input ... checked>` or w/ any value is sufficient

Reply:
I havent tested to see if the simple form options can just push through the `checked` html. I can see, but I know that it passes the checked in as an option. I can give that a bit of attention to see if I can get that working and update here.